### PR TITLE
Adding disabled behavior to links

### PR DESCRIPTION
**Pull request by Benedito0**
| - | - |
| --- | --- |
| Issue |  |
| Decisions |   |
| Animated GIF |9iPWcLHPGf |

### CODE REVIEW

File: vue/components/ui/link-platforme/link-platforme.vue [hunk 1]
```diff
@@ -30,6 +30,15 @@ a.link.hover {
...
     font-size: 12px;
 }
+
+a.link.link-disabled {
+    color: #afafba;
+    cursor: not-allowed;
+}
+
+a.link.link-disabled:hover {
+    border-color: transparent;
```
Comment (gcandal) [inline vue/components/ui/link-platforme/link-platforme.vue]: Move this to `a.link.link-disabled`, no need for the extra selector.

File: vue/components/ui/link-platforme/link-platforme.vue [hunk 2]
```diff
@@ -30,6 +30,15 @@ a.link.hover {
 a.link.link-small {
     font-size: 12px;
 }
+
+a.link.link-disabled {
+    color: #afafba;
```
Comment (gcandal) [inline vue/components/ui/link-platforme/link-platforme.vue]: Use a named color. For now you can place it in `variables.css` but its name should match what you already have in the other branch.

File: vue/components/ui/link-platforme/link-platforme.stories.js [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+        template: `
             <div>
-                <div><link-platforme v-bind:text='"This is a normal link"'></link-platforme></div>
-                <div><link-platforme v-bind:text='"This is a small link"' v-bind:size='"small"'></link-platforme></div>
+                <div><link-platforme v-bind:text='"This is a normal link"' v-bind:disabled='disabled'></link-platforme></div>
+                <div><link-platforme v-bind:text='"This is a small link"' v-bind:disabled='disabled' v-bind:size='"small"'></link-platforme></div>
             </div>
         `
-}));
+    }));
```
Comment (gcandal) [inline vue/components/ui/link-platforme/link-platforme.stories.js]: Revert

**Comment by gcandal** [comment]
Please fix the rest of the style to match Zeplin. Not just for the disable but also the color and whatever else is needed.

**Comment by joamag** [comment]
@gcandal fix the merge conflicts

File: vue/components/ui/link-platforme/link-platforme.vue [hunk 4]
```diff
@@ -17,9 +17,10 @@ a.link {
     border-bottom: 1px solid transparent;
     color: $link-color;
     cursor: pointer;
-    padding-bottom: 2px;
+    padding-bottom: 1px;
     text-decoration: none;
     transition: border-color 0.1s ease-in;
+    user-select: none;
```
Comment (joamag) [inline vue/components/ui/link-platforme/link-platforme.vue]: A link should be selectable...

**Pull request merged**